config/nonprod.tfvars

```diff
@@ -0,0 +1,10 @@
+accounts = [
+    {
+        name = "plat-ops-testuser1"
+        ips  = []
```

Comment:
if it's optional we can probably leave it out? you should be able to use the lookup function to default it to an empty array?

Reply:
There's a long open issue on support for optional properties in object variables [here.](https://github.com/hashicorp/terraform/issues/19898) Will test what works for this case.

Reply:
I would prefer to not add types to the object if that's the issue, we aren't exposing this to anyone externally so the types really give very little benefit.

Reply:
Object type was an issue and now removed.